engine/panicx/location_test.go

```diff
@@ -0,0 +1,23 @@
+package panicx_test
+
+import (
+	. "github.com/dogmatiq/testkit/engine/panicx"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("type Location", func() {
+	Describe("func String()", func() {
+		DescribeTable(
+			"it returns the expectation string",
```

Comment:
I think this should be "expected"?
```suggestion
			"it returns the expected string",
```

Reply:
Fixed on `main` :+1: